common/source/docs/common-choosing-a-ground-station.rst

```diff
@@ -65,6 +65,21 @@ Full featured and widely used GCS.
 -  `Issues List <https://github.com/ArduPilot/MissionPlanner/issues>`__
 -  `Source Code <https://github.com/ArduPilot/MissionPlanner>`__
 
+AeroGCS KEA 2.0
+---------------
+
+AeroGCS is drone management and mission planning software supporting various autopilots like Ardupilot and PX4. AeroGCS is the simplest mission planning software which helps to plan drone flights in few seconds using wizard flow. It has been so simplified that any user can learn it in few hours only. Pilot who are the actual users of drone always want simplest UI to manage it. AeroGCS main capability is its seamless integration with cloud, its unified system. Right from user registration to drone fleet tracking can be done on cloud platform. The below diagram shows overall process and capabilities. The Unified solution offers right from secure firmware to cloud business logics integration.
```

Comment:
grammatical  and English corrections, clarifications (as best that I could from the existing language)
```suggestion
AeroGCS is drone management and mission planning software supporting various autopilots like Ardupilot and PX4. AeroGCS  helps to plan drone flights in few seconds with the assistance of a built-in wizard applet. It has been designed such that any user can learn it in few hours. AeroGCS's main capability is its seamless integration with the Cloud. Upon user registration,  cloud based drone fleet tracking can be done. The diagram below shows overall system and its capabilities. This GCS provides secure firmware system for cloud-based business logistics integration.
```